src/app/parcels/parcelsTable/ParcelsPage.test.tsx

```diff
@@ -0,0 +1,237 @@
+import { processEventsDetails } from "@/app/parcels/getExpandedParcelDetails";
+import {
+    familyCountToFamilyCategory,
+    formatAddressFromClientDetails,
+    formatBreakdownOfChildrenFromFamilyDetails,
+    formatHouseholdFromFamilyDetails,
+} from "@/app/clients/getExpandedClientDetails";
+import { formatDatetimeAsDate } from "@/common/format";
+import { DbParcelRow } from "@/databaseUtils";
+import convertParcelDbtoParcelRow, {
+    processLastStatus,
+    ProcessParcelDataResult,
+} from "@/app/parcels/parcelsTable/convertParcelDBtoParcelRow";
+import { CongestionChargeDetails, ParcelsTableRow } from "@/app/parcels/parcelsTable/types";
+import { expect, it } from "@jest/globals";
+import "@testing-library/jest-dom/jest-globals";
```

Comment:
This line isn't actually required here, since these are really just unit tests not component tests and we don't need a DOM